docs/application/dotnet/guides/nui/nui-components/Tab.md

```diff
@@ -7,46 +7,48 @@ A tab can contain one or more TabItem with text, usually used as a menu label. A
 ![Tab](./media/tab.png) ![Tab](./media/tab2.png)

```

Comment:
Please remove the space before the starting the sentences from Line no. 1 to 5. Also Please put one line difference between all these lines as:

# Tab
Tab is a common component and it can be used as a menu label.  

A tab makes it easy to explore and switch between different views or functional aspects of an application or to browse categorized data sets.  

You can handle a tab by adding, inserting, or deleting a TabItem. A tab can contain one or more TabItem with text, usually used as a menu label. A TabItem can have different lengths.